lib/simulator/SimulatorLightController.hpp

```diff
@@ -12,35 +11,22 @@
 /*
  * Interface class for controlling the output lights.
  */
-class SimulatorLightController {
+class SimulatorLightController : public FastLEDSimulator<96> {
```

Comment:
Rather than requiring users of this class to call `Init()` and `Close()`, I would prefer if the constructor did the initialization and the destructor did any necessary cleanup. It also looks like our users don't need to use this class polymorphically or generally care that it derives from `FastLEDSimulator<96>`, so I would prefer to use `private` inheritance instead of `public`, and just have a `using FastLEDSimulator<96>::Run;` in the class body. This helps keep our interface small and include only what we actually need.

Reply:
TIL about this. Done.